wee_alloc/Cargo.toml

```diff
@@ -26,3 +26,6 @@ use_std_for_test_debugging = []
 [target.'cfg(all(unix, not(target_arch = "wasm32")))'.dependencies.libc]
 default-features = false
 version = "0.2"
+
+[target.'cfg(any(windows, target_os = "linux", target_os = "macos"))'.dependencies.mmap-alloc]
```

Comment:
I don't think we can support windows until we have a version of `imp::Exclusive` that uses `CRITICAL_SECTION` or whatever.

Is there a reason you did `target_os = "linux"` and `target_os = "macos"` instead of `unix` here and elsewhere?

Reply:
Yeah, its probably true that the old `mmap` implementation wasn't strictly 100% portable.

Thanks again!

Reply:
np!